I got one failure
```
======================================================================
ERROR: testStored (test.test_zipfile64.TestsWithSourceFile)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/nix-build-pypy-5.1.1.drv-0/pypy-pypy-b0a649e90b66/lib-python/2.7/test/test_zipfile64.py", line 83, in testStored
    self.zipTest(f, zipfile.ZIP_STORED)
  File "/tmp/nix-build-pypy-5.1.1.drv-0/pypy-pypy-b0a649e90b66/lib-python/2.7/test/test_zipfile64.py", line 56, in zipTest
    zipfp.writestr("testfn%d" % num, self.data)
  File "/tmp/nix-build-pypy-5.1.1.drv-0/pypy-pypy-b0a649e90b66/lib-python/2.7/zipfile.py", line 1248, in writestr
    self.fp.write(bytes)
IOError: [Errno 28] No space left on device: '<fdopen>'

----------------------------------------------------------------------
Ran 4 tests in 186.267s

FAILED (errors=1)
Traceback (most recent call last):
  File "/tmp/nix-build-pypy-5.1.1.drv-0/pypy-pypy-b0a649e90b66/pypy/tool/pytest/run-script/regrverbose.py", line 14, in <module>
    indirect_test()
  File "/tmp/nix-build-pypy-5.1.1.drv-0/pypy-pypy-b0a649e90b66/lib-python/2.7/test/test_zipfile64.py", line 156, in test_main
    run_unittest(TestsWithSourceFile, OtherTests)
  File "/tmp/nix-build-pypy-5.1.1.drv-0/pypy-pypy-b0a649e90b66/lib-python/2.7/test/test_support.py", line 1450, in run_unittest
    _run_suite(suite)
  File "/tmp/nix-build-pypy-5.1.1.drv-0/pypy-pypy-b0a649e90b66/lib-python/2.7/test/test_support.py", line 1404, in _run_suite
    raise TestFailed(err)
TestFailed: Traceback (most recent call last):
  File "/tmp/nix-build-pypy-5.1.1.drv-0/pypy-pypy-b0a649e90b66/lib-python/2.7/test/test_zipfile64.py", line 83, in testStored
    self.zipTest(f, zipfile.ZIP_STORED)
  File "/tmp/nix-build-pypy-5.1.1.drv-0/pypy-pypy-b0a649e90b66/lib-python/2.7/test/test_zipfile64.py", line 56, in zipTest
    zipfp.writestr("testfn%d" % num, self.data)
  File "/tmp/nix-build-pypy-5.1.1.drv-0/pypy-pypy-b0a649e90b66/lib-python/2.7/zipfile.py", line 1248, in writestr
    self.fp.write(bytes)
IOError: [Errno 28] No space left on device: '<fdopen>'
```

--- a/pkgs/development/interpreters/pypy/default.nix
+++ b/pkgs/development/interpreters/pypy/default.nix
@@ -73,8 +73,9 @@ let
        # disable socket because it has two actual network tests that fail
        # disable test_urllib2net, test_urllib2_localnet, and test_urllibnet because they require networking (example.com)
        # disable test_ssl because no shared cipher' not found in '[Errno 1] error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure
+       # disable test_zipfile64 because it causes ENOSPACE
        # disable test_epoll because of invalid arg, should be fixed in as of version 5.1.2
-      ./pypy-c ./pypy/test_all.py --pypy=./pypy-c -k 'not ( test_ssl or test_urllib2net or test_urllibnet or test_urllib2_localnet or test_socket or test_shutil or test_epoll )' lib-python
+      ./pypy-c ./pypy/test_all.py --pypy=./pypy-c -k 'not ( test_ssl or test_urllib2net or test_urllibnet or test_urllib2_localnet or test_socket or test_shutil or test_zipfile64 or test_epoll )' lib-python
     '';
 
     installPhase = ''

